pkg/devfile/adapters/common/command.go

```diff
@@ -39,8 +40,15 @@ func getCommandFromDevfile(data data.DevfileData, groupType common.DevfileComman
 	}
 
 	for _, command := range commands {
-		// exec.Group is a pointer, to avoid null pointer
-		if command.Exec.Group != nil && command.Exec.Group.Kind == groupType {
+		if command.Composite != nil && command.Composite.Group != nil && command.Composite.Group.Kind == groupType {
```

Comment:
Can the if and else if be combined? since their blocks are doing the same thing and its only the condition thats different

Reply:
Can we add some comp cmd unit test cases for this func?

Reply:
Yeah, if I had a helper function to the Command type to receive the Kind, it can be simplified

Reply:
Done